.chloggen/3149-add-must-gather.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: operator
```

Comment:
there is no operator component. Which CRs can be debugged with the must gather?

Reply:
`OpenTelemetryCollector` and `Instrumentation`. I'll add them